src/app/shared/models/vehicle.ts

```diff
@@ -0,0 +1,13 @@
+import { Transmission } from "@app/shared/models/transmission";
+
+export class Vehicle {
```

Comment:
Nie musisz nic zmieniać, tylko informacyjnie, ale tutaj nie musisz się już trzymać schemy, którą dostajesz z BE. Może sobie dzielić jak tylko chcesz :p

Reply:
a jak chce tak jak jest na BE? :slightly_frowning_face: 
(wezmę to pod uwagę na przyszłość) :smile: